warehouse/upload.go

```diff
@@ -68,43 +68,6 @@ type uploadStateT struct {
 	nextState  *uploadStateT
 }
 
-type Upload struct {
```

Comment:
Since this is removed now, We can probably remove the redundant type conversions.
https://github.com/rudderlabs/rudder-server/blob/97c27df038f333dd7f0a1bf75c9f8dc12ca81b75/warehouse/warehouse.go#L798
https://github.com/rudderlabs/rudder-server/blob/97c27df038f333dd7f0a1bf75c9f8dc12ca81b75/warehouse/warehouse.go#L798